BO4E/COM/Zeitfenster.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Text.Json.Serialization;
+
+using Newtonsoft.Json;
+
+using ProtoBuf;
+
+namespace BO4E.COM
+{
+    /// <summary>
+    /// Ein Zeitfenster mit Start- und Endzeiten mit <see cref="TimeOnly"/> als Typ.
+    /// </summary>
+    [ProtoContract]
+    public class Zeitfenster : COM
+    {
+        /// <summary>inklusive Startzeit; z.B. 8 Uhr</summary>
+        [JsonProperty(Required = Required.Default, Order = 3, PropertyName = "startzeit")]
+        [JsonPropertyName("startzeit")]
+        [JsonPropertyOrder(3)]
+        [ProtoMember(3)]
+        public TimeOnly? Startzeit { get; set; }
+
+        /// <summary>exklusive Endzeit (z.B. 17:00 Uhr)</summary>
+        [JsonProperty(Required = Required.Default, Order = 4, PropertyName = "endzeit")]
+        [JsonPropertyName("endzeit")]
+        [JsonPropertyOrder(4)]
+        [ProtoMember(4)]
+        public TimeOnly? Endzeit { get; set; }
+
+        /// <summary>
+        /// Erstellt eine neue Instanz von Zeitfenster mit spezifischen Start- und Endzeiten.
+        /// </summary>
+        /// <param name="startzeit">inklusive Startzeit</param>
+        /// <param name="endzeit">exklusive Endzeit</param>
+        public Zeitfenster(TimeOnly startzeit, TimeOnly endzeit)
+        {
+            Startzeit = startzeit;
+            Endzeit = endzeit;
+        }
+
+        /// <summary>
+        /// Parameterloser Konstruktor für Serialisierungszwecke
+        /// </summary>
+        public Zeitfenster() { }
+
+        /// <summary>
+        /// Erstellt eine neue Instanz von Zeitfenster basierend auf einem String im Format "HHMMHHMM".
```

Comment:
Wollen wir das umwandeln zu HHMMHHMM wirklich in der library machen oder dann erst im converter?

Reply:
Habe eure Unterhaltung hierzu gelesen und das schein ja alles schon gut durchdacht zu sein, dann gerne so lassen.👍